Dockerfile

```diff
@@ -2,10 +2,10 @@ ARG BASE_IMAGE_FULL
 ARG BASE_IMAGE_MINIMAL
 
 # Build node feature discovery
-FROM golang:1.17.13-buster as builder
+FROM golang:1.17.13 as builder
 
 # Build and install the grpc-health-probe binary
-RUN GRPC_HEALTH_PROBE_VERSION=v0.4.15 && \
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.17 && \
```

Comment:
You can upgrade to [v0.4.18](https://github.com/grpc-ecosystem/grpc-health-probe/releases/tag/v0.4.18) of grpc-health-probe 

The changes there include addition of dependabot followed by dependency upgrades https://github.com/grpc-ecosystem/grpc-health-probe/commits/master


They have already done this in master branch upstream https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/Dockerfile#L9

Reply:
ok will do that